mne/digitization/base.py

```diff
@@ -103,3 +103,28 @@ def __eq__(self, other):  # noqa: D105
             return False
         else:
             return all([ss == oo for ss, oo in zip(self, other)])
+
+    def plot(self, trans=None, subject=None, subjects_dir=None):
+        """Plot head, sensor, and source space alignment in 3D.
+
+        Parameters
+        ----------
+        trans : str | 'auto' | dict | None
```

Comment:
docstring does not cover the "auto" and "dict" usage. Also this docstring should be made
a template in doc.py to be shared among functions that need a trans param